### PR TITLE
BCDA-3769 Updated stylesheets for static site glow up

### PR DESCRIPTION
**Pull request by PatKelsh**
### Fixes [BCDA-3769](https://jira.cms.gov/browse/BCDA-3769)

Migrated over stylesheets from DPC static site. Now using node package of CMS design sytem

### Proposed Changes

* Added CMS design system
* Updated jekyll config for sass precompile
* Removed old stylesheets

### Security Implications

- [x] new software dependencies

The new CMS design system node installation

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

- [x] no PHI/PII is affected by this change

### Acceptance Validation

Insert this into the top of the `_layouts/home.html` file and start the server. Should see a spinner at the top of the home page.
```
<div class="ds-l-row">
  <span class="ds-c-spinner" aria-valuetext="Loading" role="progressbar"></span>
</div>
```

### Feedback Requested

Do we want to go in this direction and copy DPC completely or would like to keep the old way of just copying the stylesheets from the CMS design system into the `_assests` directory?

### CODE REVIEW

File: assets/css/static-main.scss [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+    text-decoration: none;
+    color: #fff;
+    transition: all .3s ease
+}
+
+.dropdown-content a {
+    text-decoration: none;
+    display: block;
+}
+*/
```
Comment (jeffcorpuz) [inline assets/css/static-main.scss:144]: will this be added later? would you mind commenting out why this is commented out?

File: assets/css/static-main.scss [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+    padding: 1%;
+}
+
+.submenu:active,
+.submenu:focus,
+.submenu:hover {
+    background-color: #fff;
+    color: #339392
+}
+*/
```
Comment (jeffcorpuz) [inline assets/css/static-main.scss:199]: likewise, can you comment why this is commented out?

**Review by mtrang1263 — CHANGES_REQUESTED** [review]
A couple notes:
1. Can we update the PR title to give a summary of these changes?
2. We should switch the base branch from `master` to `static-site-glowup`?

**Review by mtrang1263 — APPROVED** [review]
LGTM!

**Pull request merged**